code/defines/obj.dm

```diff
@@ -11,8 +11,8 @@
 		switch(alert("Travel back to ss13?",,"Yes","No"))
 			if("Yes")
 				if(user.z != src.z)	return
-				user.loc.loc.Exited(user)
-				user.loc = pick(latejoin)
+				user.loc.loc.Exited(user) //what the fuck is this

```

Comment:
This is something that should be no longer necessary. forceMove ensures in this being proper.

Reply:
So how should this look? just user.ForceMove(loc) ?

Reply:
Yes.